app/bootstrap.php

```diff
@@ -24,6 +24,7 @@
 $app->register( new SessionServiceProvider() );
 $app->register( new RoutingServiceProvider() );
 $app->register( new TwigServiceProvider() );
+$ffFactory->setTwigEnvironment( $app['twig'] );
```

Comment:
This is a slightly surprising location to me. Before this file just set up $app, now it also has a side effect on the factory. 

We already have such side effects based on $app in index.dev.php, so the same could be done for this. However you'd need to put it in both entry points, and possibly in the test environment thing that includes this file as well.